fedimint-client/src/client.rs

```diff
@@ -1173,6 +1174,10 @@ impl Client {
             task_group
                 .clone()
                 .spawn_cancellable("refresh_common_api_version_static", async move {
+                    // Delay making any network connections, just in case this is a short-running
+                    // invocation of the client (e.g. `fedimint-cli info`), so the attempt probably
+                    // won't succeed anyway.
+                    sleep(Duration::from_secs(1)).await;
```

Comment:
Should we try putting such constants into constants with a name? I guess here the comment is enough, but often I find is useful to have a name for the delay.